src/bindings/python/tests/test_runtime/test_sync_infer_request.py

```diff
@@ -15,7 +15,7 @@
     Core,
     CompiledModel,
     InferRequest,
-    AsyncInferQueue,
+    # AsyncInferQueue,
```

Comment:
Could you please remove it completely? We tend to avoid having commented out code.

Reply:
And remove/uncomment all commented tests